app/routes/api.getGitHubRelease.$owner.$repository.tsx

```diff
@@ -0,0 +1,28 @@
+import { LoaderFunction, json } from "@remix-run/cloudflare";
+
+const CACHE_DURATION = 60 * 60; // 1 hour
+
+export const loader: LoaderFunction = async ({ params }) => {
+    const repository: string = `${params.owner}/${params.repository}`.toLowerCase();
```

Comment:
Should we have a check for the repository name and owner? As it is written now in theory bad actor can use this to check random repositories on Github through our account.

Reply:
For a validation check it is fine to have a hard coded list, there isn't really a choice, if you don't want to check all the repositories at least limit the owners with a list of valid owners we accept, it should be shorter than full owner+repo list and sufficient for our purposes.